ui/packages/consul-ui/tests/helpers/type-to-url.js

```diff
@@ -5,9 +5,11 @@ export default function(type) {
       requests = ['/v1/catalog/datacenters'];
       break;
     case 'service':
-    case 'instance':
       requests = ['/v1/internal/ui/services', '/v1/health/service/'];
       break;
+    case 'instance':
+      requests = ['/v1/health/service/'];
+      break;
```

Comment:
During acceptance tests when you write:

Given 2 _instance_ models from yaml:
\---
\- Service
...etc
\---

This means that it will only overwrite the mock data received from the health endpoint, not both the service listing _and_ the health endpoint. There are a few legacy/historical reasons why this happened this way, but this is more correct.